tests/playwright-test/expect-to-pass.spec.ts

```diff
@@ -260,3 +260,35 @@ test('should give priority to timeout parameter over timeout in config file', as
   4 |         await test.expect(() => {
   `.trim());
 });
+
+test('should respect intervals in config file when intervals parameter is not passed', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'playwright.config.js': `module.exports = { expect: { toPass: { intervals: [100, 200] } } }`,
+    'a.spec.ts': `
+      import { test, expect } from '@playwright/test';
+      test('should fail', async () => {
+        await test.expect(() => {
+          expect(1).toBe(2);
+        }).toPass();
+      });
+    `
+  });
+  expect(result.exitCode).toBe('TODO');
```

Comment:
Replace TODO with a status code?

Reply:
Yes of course!  I wanted it to fail because I didn't have time to implement it.  I will try to do it in the next few days or weekend